examples/api/README.md

```diff
@@ -26,7 +26,7 @@ This will install all dependencies.
 
 `npm start`
 
-This will call babel to transpile app.js and run it via node. 
+This will call babel to transpile app.tsx and run it via node.
```

Comment:
Babel has been removed.

Reply:
oops, this example actually still using it. I must have picked up this ext in a search and replace. I only updated the main example (esp-js-react-agile-board) to TypeScript.